tools/libfrencutils/create_xgrid.c

```diff
@@ -29,7 +29,8 @@
 #define EPSLN8            (1.e-8)
 #define EPSLN30           (1.0e-30)
 #define EPSLN10           (1.0e-10)
-
+#define R2D (180/M_PI)
```

Comment:
Constants like R2D and PTI were removed from here; they are in tools/libfrencutils/constant.h. Another reason to start with a copy of the baseline file.

Reply:
I did start with the baseline file and went through all these changes myself. Since the changes are from another repository there's no easy way of telling what was done in between now and the hackathon, especially since the entire file is formatted differently.